tools/cli/commands.go

```diff
@@ -857,8 +862,15 @@ func convertDescribeWorkflowExecutionResponse(resp *shared.DescribeWorkflowExecu
 			ActivityID:             pa.ActivityID,
 			ActivityType:           pa.ActivityType,
 			State:                  pa.State,
-			HeartbeatDetails:       common.StringPtr(string(pa.HeartbeatDetails)),
-			LastHeartbeatTimestamp: common.StringPtr(convertTime(pa.GetLastHeartbeatTimestamp(), false)),
+			ScheduledTimestamp:     timestampPtrToStringPtr(pa.ScheduledTimestamp, false),
+			LastStartedTimestamp:   timestampPtrToStringPtr(pa.LastStartedTimestamp, false),
+			LastHeartbeatTimestamp: timestampPtrToStringPtr(pa.LastHeartbeatTimestamp, false),
+			Attempt:                pa.Attempt,
+			MaximumAttempts:        pa.MaximumAttempts,
```

Comment:
It seems like we want to do some checks here to see if these are zero. If they are zero then just use some string "not set" that way its clear that entering 0 was a not a bug but rather that there just is no termination case based on that condition.

Reply:
I agree with you. The problem here is how these information are printed. Currently we are using json.MarshalIndent to do that. To add that check, we basically need to do all the formatting ourselves. That's also why at sever side, I carefully ensure that a nil pointer is returned for these fields instead of a pointer to value 0.

Reply:
Sounds reasonable.